alcs-frontend/src/app/features/application/applicant-info/application-details/parcel/parcel.component.html

```diff
@@ -69,10 +69,54 @@ <h4 [id]="parcel.uuid">Parcel #{{ parcelInd + 1 }}: Parcel and Owner Information
         <a (click)="openFile(parcel.certificateOfTitle)">{{ parcel.certificateOfTitle.fileName }}</a>
       </div>
     </div>
-    <div
-      class="full-width owner-information"
-      [ngClass]="{ 'crown-owner-information': parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN }"
-    >
+    <ng-container *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+      <h5 *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">Government Parcel Contact</h5>
+      <ng-container *ngIf="parcel.owners[0]">
+        <div class="subheading2 grid-1">First Name</div>
+        <div class="grid-double">
+          {{ parcel.owners[0].firstName }}
+          <app-no-data *ngIf="!parcel.owners[0].firstName"></app-no-data>
+        </div>
+        <div class="subheading2 grid-1" *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+          Last Name
+        </div>
+        <div class="grid-double">
+          {{ parcel.owners[0].lastName }}
+          <app-no-data *ngIf="!parcel.owners[0].lastName"></app-no-data>
+        </div>
+        <div class="subheading2 grid-1" *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+          Ministry or Department
+        </div>
+        <div class="grid-double">
+          {{ parcel.owners[0].organizationName }}
+          <app-no-data *ngIf="!parcel.owners[0].organizationName"></app-no-data>
+        </div>
+        <div class="subheading2 grid-1" *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+          Phone
+        </div>
+        <div class="grid-double">
+          {{ parcel.owners[0].phoneNumber }}
+          <app-no-data *ngIf="!parcel.owners[0].phoneNumber"></app-no-data>
+        </div>
+        <div class="subheading2 grid-1" *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+          Email
+        </div>
+        <div class="grid-double">
+          {{ parcel.owners[0].email }}
+          <app-no-data *ngIf="!parcel.owners[0].email"></app-no-data>
+        </div>
+        <div class="subheading2 grid-1" *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
+          Crown Type
+        </div>
+        <div class="grid-double">
+          {{ parcel.owners[0].crownLandOwnerType === 'provincial' ? 'Provincial Crown' : '' }}
+          {{ parcel.owners[0].crownLandOwnerType === 'federal' ? 'Federal Crown' : '' }}
+          <app-no-data *ngIf="!parcel.owners[0].crownLandOwnerType"></app-no-data>
+        </div>
+      </ng-container>
+      <app-no-data *ngIf="!parcel.owners[0]"></app-no-data>
+    </ng-container>
+    <div class="full-width owner-information" *ngIf="parcel.ownershipTypeCode !== PARCEL_OWNERSHIP_TYPES.CROWN">
       <div class="subheading2" *ngIf="parcel.ownershipTypeCode !== PARCEL_OWNERSHIP_TYPES.CROWN">Land Owner(s)</div>
       <div class="subheading2" *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">Crown Contact(s)</div>
```

Comment:
Delete this, this will never run because of wrapper *ngIf

Same for Ministry/Department